core-bundle/contao/templates/forms/form_captcha.html5

```diff
@@ -22,7 +22,9 @@
       <label for="ctrl_<?= $this->id ?>_hp">Do not fill in this field</label>
       <input type="text" name="<?= $this->name ?>_name" id="ctrl_<?= $this->id ?>_hp" value="">
     </div>
-    <script src="<?= $this->getAjaxUrl() ?>" data-id="<?= $this->id ?>" data-name="<?= $this->name ?>" async></script>
-  <?php endif ?>
+    <?php if (!$this->isBackendRequest()): ?>
```

Comment:
Not sure if a method is the best solution here? We could also add an `->asEditorView` variable to all templates and use it here.

Reply:
👍 for `->asEditorView`

Reply:
Changed in 933117ccd865d0d10499e077f41a9804282dd80c. Since it is a variable, I had to add it both to the `Template` and the `Widget` class. If it were a method, we could add it to the `TemplateTrait` instead. What do you prefer?

Reply:
+1 for how it is right now 👍